File: Python/Logic-2/no_teen_sum.py
def no_teen_sum(a, b, c):
  sum = fix_teen(a) + fix_teen(b) + fix_teen(c)
  return sum

def fix_teen(n):
  if (n != 16 and n != 15):
    if (n <= 19 and n >= 13):
      return 0
  return n

File: Java/Logic-2/noTeenSum.java
public int noTeenSum(int a, int b, int c) {
  return fixTeen(a) + fixTeen(b) + fixTeen(c);
}

public int fixTeen(int n) {
  if (n != 15 && n != 16 && n > 12 && n < 20) return 0;
  return n;
}
